makefile

```diff
@@ -443,12 +478,45 @@ define clone_repo
     fi
 endef
 
+# download: header-only, no compile needed...
 $(XSIMD_DIR)/include/xsimd/xsimd.hpp:
 	mkdir -p $(DEPS_ROOT)
-	@echo "Checking xsimd external dependency..."
+	@echo "Checking XSIMD external dependency..."
 	$(call clone_repo,$(XSIMD_URL),$(XSIMD_VERSION),$(XSIMD_DIR))
 	@echo "xsimd installed in deps/xsimd"
 
+# download DUCC... (an empty target just used to track if installed)
+$(DUCC_COOKIE):
```

Comment:
this can go wrong if the user has a wrong version of ducc

Reply:
Without getting into git submodules, I don't see how we can test the user's ducc version #. So I see no way to automate this. The user simply has to make setupclean and try again.